### DISCUSSION TITLE
How to generate server less debug apk?

### DISCUSSION THREAD

**Discussion by Ashik55** [RocketChat/Rocket.Chat.ReactNative#3027] [Q&A] [answered]
I want to share the apk with my team-mates but how can I generate apk which will run without server.
This command is not working for rockect chat & shows Unable to read script &metro server not found :

"react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res"

**Comment by diegolmello** [comment, marked as answer]
You need to build release https://reactnative.dev/docs/signed-apk-android
